improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -615,16 +615,25 @@ def process(self, forecast_probabilities, no_of_percentiles=None,
         return forecast_at_percentiles
 
 
-class FromMeanAndVariance():
+class ConvertLocationAndScaleParameters():
     """
     Base Class to support the plugins that compute percentiles and
-    probabilities from the mean and variance.
+    probabilities from the location and scale parameters.
     """
 
     def __init__(self, distribution="norm", shape_parameters=None):
         """
         Initialise the class.
 
+        In order to construct percentiles or probabilities from the location
+        or scale parameter, the distribution for the resulting output needs
+        to be selected. For use with the outputs from EMOS, where it has been
+        assumed that the outputs from minimising the CRPS follow a particular
+        distribution, then the same distribution should be selected, as used
+        for the CRPS minimisation. The conversion to percentiles and
+        probabilities from the location and scale parameter relies upon
+        functionality within scipy.stats.
+
```

Comment:
Whilst not part of this work, should we be recording the distribution assumed in the coefficients cube in minimising the CRPS so that the coefficients cannot be applied inappropriately?

Reply:
We could do this, so rather than pass the distribution into the apply-emos-coefficients CLI, add metadata to record the distribution. We can have a chat about this.

Reply:
Let's have that chat sometime.

Reply:
Would that mean you wouldn't need to specify the distribution or shape parameters here? It would just take them from the coefficients cube?

Reply:
Created a new ticket IMPRO-1578 to cover this work. We can talk about it at some point.